src/components/Panel/PanelHeader.tsx

```diff
@@ -0,0 +1,38 @@
+import React, { FC } from 'react';
+
+import { mergeClasses } from '../../shared/utilities';
+
+import { EventType } from './Panel.types';
+
+import { SystemUIButton } from '../Button';
+import { IconName } from '../Icon';
+
+import styles from './panel.module.scss';
+
+interface PanelHeaderProps {
+    onClose?: (e: EventType) => void;
+    title?: string;
+}
+
+export const PanelHeader: FC<PanelHeaderProps> = ({ onClose, title }) => {
+    const headerTitle: string = mergeClasses(['header4', styles.headerTitle]);
+
+    return (
+        <div className={styles.logoGradientHeaderWrapper}>
+            <div className={headerTitle}>{title}</div>
+            <div>
+                {!!onClose && (
+                    <SystemUIButton
+                        iconProps={{
+                            path: IconName.mdiClose,
+                            color: 'var(--white-color)',
+                        }}
+                        ariaLabel={'Close'}
+                        onClick={onClose}
+                        transparent
+                    />
+                )}
```

Comment:
add other buttons as well,

Reply:
@ychhabra-eightfold we do not have other icons at present in the library and considering the time crunch, maybe we can take this in later point? Currently we only need a close button for sms and whatsapp panel use case.

Reply:
@mfazil-eightfold it only takes a moment to add the icon paths to https://github.com/EightfoldAI/octuple/blob/main/src/components/Icon/mdi.ts

Reply:
The comment at the top of mdi.ts explains how to do this.

Reply:
If the old component uses FA, the mapping of FA to MDI is in the figma in the comment

Reply:
@dkilgore-eightfold @ychhabra-eightfold i've added those changes kindly check.